authorization/resource/resource_type.go

```diff
@@ -77,6 +78,36 @@ func (m *GormResourceTypeRepository) Load(ctx context.Context, id uuid.UUID) (*R
 	return &native, errs.WithStack(err)
 }
 
+func (m *GormResourceTypeRepository) LookupOrCreate(ctx context.Context, name string) (*ResourceType, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "resource_type", "lookupOrCreate"}, time.Now())
+
+	var native *ResourceType
+	err := m.db.Table(m.TableName()).Where("name = ?", name).Find(&native).Error
+
+	if err == gorm.ErrRecordNotFound {
+		native = &ResourceType{
+			ResourceTypeID: uuid.NewV4(),
+			Name:           name,
+		}
+
+		err := m.db.Create(native).Error
+
+		if err != nil {
+			log.Error(ctx, map[string]interface{}{
+				"resource_type_id": native.ResourceTypeID,
+				"err":              err,
+			}, "unable to create the resource type")
+			return nil, errs.WithStack(err)
+		}
+		log.Debug(ctx, map[string]interface{}{
+			"resource_type_id": native.ResourceTypeID,
```

Comment:
let's add "name": name too

Reply:
I've added the "name" fields to the logging.